server/migrations/20240205152029_add_user_id_to_tables.sql

```diff
@@ -0,0 +1,5 @@
+-- Add migration script here
+
+ALTER TABLE pipes ADD COLUMN user_id TEXT DEFAULT '';
```

Comment:
Are any of these used in JOIN's? If so we should keep in the back of our minds that they'll probably need an index, eventually.

Reply:
not used in any joins yet, but they are used in lots of where queries, which can definitely benefit from an index. added!

Reply:
Thank you!